src/Modules/SimplCommerce.Module.Catalog/Models/Product.cs

```diff
@@ -155,6 +168,33 @@ public Product Clone()
                 });
             }
 
+            foreach (var media in Medias)
+            {
+                product.AddMedia(new ProductMedia
+                {
+                    MediaId = media.MediaId
+                });
+            }
+
+            foreach (var option in OptionValues)
```

Comment:
I'm not sure whether we should clone the product options and product variation (product link with ProductLinkType.Super).

This behavior can also have side effect when create a new product with variations. This clone method is called for each variation

Reply:
okay let me remove the product options and variations clone.